Sources/TuistHasher/GraphContentHasher.swift

```diff
@@ -48,6 +60,30 @@ public struct GraphContentHasher: GraphContentHashing {
         let hashedTargets: ThreadSafe<[GraphHashedTarget: String]> = ThreadSafe([:])
         let hashedPaths: ThreadSafe<[AbsolutePath: String]> = ThreadSafe([:])
 
+        var additionalStrings = additionalStrings
+        if let workspacePath = try await fileSystem.glob(directory: graph.path, include: ["*.xcworkspace"]).collect().first {
```

Comment:
Isn't this logic assuming non-generated projects?

Reply:
If you're using `Tuist/Package.swift`, then we don't need to hash the `Package.resolved` and it wouldn't even be available. We _should_ hash the `.package.resolved` in the root, though. I'll add that.